src/checkPrime.js

```diff
@@ -0,0 +1,10 @@
+// Write a JavaScript function to check if a given number is prime or not.
+
+function checkPrime(num) {
+  if (num < 2) return false;
+  if (num === 2) return true;
+  for (let i = 2; i <= Math.sqrt(num); i++) {
+    if (num % i === 0) return false;
+  }
+  return true;
+}
```

Comment:
Nice move on using the loop and the square root 💯 . It decreases the input size which improves your solution efficiency.